ci-operator/step-registry/mirror-images/by-oc-image-mirror/mirror-images-by-oc-image-mirror-commands.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+new_pull_secret="${SHARED_DIR}/new_pull_secret"
+qe_image="/tmp/mirror-images-list.yaml"
+
+# private mirror registry host
+# <public_dns>:<port>
+MIRROR_REGISTRY_HOST=`head -n 1 "${SHARED_DIR}/mirror_registry_url"`
+if [ ! -f "${SHARED_DIR}/mirror_registry_url" ]; then
+    echo "File ${SHARED_DIR}/mirror_registry_url does not exist."
+    exit 1
+fi
+echo "MIRROR_REGISTRY_HOST: $MIRROR_REGISTRY_HOST"
+
+# Set mirror registry port 6001 for quay.io/openshifttest
+MIRROR_PROXY_REGISTRY_QUAY=`echo "${MIRROR_REGISTRY_HOST}" | sed 's/5000/6001/g' `
```

Comment:
The proxy feature is enabled on 6001 registry server, I'm not sure it works on C2S/SC2S env.